src/models/quote.js

```diff
@@ -65,7 +67,7 @@ function * getQuote (params, config, routeBuilder, balanceCache) {
     quote.destinationExpiryDuration,
     quote.minMessageWindow, config)
 
-  return {
+  return omitUndefined({
```

Comment:
Why is the call to omitUndefined(...) necessary? Is there any scenario where the argument to this function can be undefined?